packages/fiori-elements-writer/templates/v4/feop/add/webapp/Component.js

```diff
@@ -0,0 +1,36 @@
+sap.ui.define(
+    ["<%- app.baseComponent %>"],
+    function (UIComponent) {
```

Comment:
Since this file is anyway specific to feop, shouldn't you have the import predefined as sap/fe/core/AppComponent and the name of the parameter to be AppComponent

Reply:
Could do, but I'd prefer a consistent use of templating across all template types to ease updates in future. i.e. the component name is only defined in one place in this case.

Reply:
at least replace UIComponent by AppComponent in this case then :)

Reply:
How about something generic instead `Component` ;)